stories/assets/scss/_base.scss

```diff
@@ -39,42 +39,31 @@
   }
 }
 
-.scrollbar-horizontal {
+.custom-scrollbar {
+  scroll-behavior: smooth;
   overflow-x: auto;
-
-  &::-webkit-scrollbar-track {
-    border-bottom: 1px solid $color-black;
-    margin-right: -1px;
-  }
+  overflow-y: auto;
 
   &::-webkit-scrollbar {
+    width: 4px;
     height: 4px;
-    margin-right: 2px;
-  }
-
-  &::-webkit-scrollbar-thumb {
-    background-color: $color-black;
   }
-}
-
-.scrollbar-vertical {
-  overflow-y: auto;
 
   &::-webkit-scrollbar-track {
-    border-right: 1px solid $color-black;
-    margin-bottom: -1px;
+    background: $color-black;
+    border: 1px solid $color-white;
```

Comment:
@harishkumar-srijan Also add the following property here
```
background-clip: padding-box;
```
The above property will make sure that the background will only extend to the inside edges of the border.

Reply:
feedback done